webrtc.html

```diff
@@ -4484,8 +4493,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <li>(<code>connected</code>, finished all checks):
           <code>completed</code></li>
 
-          <li>(<code>completed</code>, lost connectivity):
-          <code>disconnected</code></li>
```

Comment:
The "lost connectivity" transition to "disconnected" is still possible, so I don't think this needs to be removed.